cmd/build_metadata/update.sh

```diff
@@ -8,27 +8,17 @@ export LC_ALL=C
 new=$(mktemp -t emailverifierXXX)
 
 # 1. update disposable domains meta databases
-curl --silent https://raw.githubusercontent.com/tompec/disposable-email-domains/main/index.json | jq -r '.[]' > $new
-
-tmp=$(mktemp -t emailverifierXXX)
-cat $new ./disposable.txt \
-    | sed '/^$/d' \
-    | sed '/./,$!d' \
-    | sed -e 's/^ *//' -e 's/ *$//' \
-    | awk '{print tolower($0)}' \
-    | sort \
-    | uniq  > $tmp
```

Comment:
Why do we remove this deduplicate behavior?

Reply:
The new behavior use the upstream data directly, no need to do the deduplicate action.

Reply:
I see. To ensure I understand your point correctly. Do you mean the upstream data has been already deduplicated? if yes, what about the free domains?

Reply:
Yes, the upstream data is already deduplicated. 
For free domains, except the main datasource `Kikobeats/free-email-domains`, there are another sources in `free_domain_sources.txt`, so the deduplicate action is needed.